app/Models/LogDAO.php

```diff
@@ -12,9 +12,9 @@ public static function logPath(?string $logFileName = null): string {
 	public static function lines(?string $logFileName = null): array {
 		$logs = [];
 		$handle = @fopen(self::logPath($logFileName), 'r');
-		if ($handle) {
+		if ($handle !== false) {
 			while (($line = fgets($handle)) !== false) {
-				if (preg_match('/^\[([^\[]+)\] \[([^\[]+)\] --- (.*)$/', $line, $matches)) {
+				if (is_int(preg_match('/^\[([^\[]+)\] \[([^\[]+)\] --- (.*)$/', $line, $matches))) {
```

Comment:
This looks wrong.
Anyway, let's refocus this PR on solely where we should have `declare(strict_types=1);`